src/platform/packages/shared/kbn-esql-ast/scripts/generate_function_definitions.ts

```diff
@@ -304,13 +312,14 @@ function printGeneratedFunctionsFile(
     alias: ${alias ? `['${alias.join("', '")}']` : 'undefined'},
     signatures: ${JSON.stringify(signatures, null, 2)},
     locationsAvailable: [${locationsAvailable.join(', ')}],
+    ${license ? `license: '${license}',` : ''}${license ? '' : ''}
     validate: ${functionDefinition.validate || 'undefined'},
     examples: ${JSON.stringify(functionDefinition.examples || [])},${
       customParametersSnippet
         ? `\ncustomParametersSnippet: ${JSON.stringify(customParametersSnippet)},`
         : ''
     }
-}`;
+}`.replace(/\n\s+\n/g, '\n'); // Remove empty lines like license: '
```

Comment:
Is it necessary? You can default to undefined if there is no license info coming from ES

Reply:
I wanted to maintain the current behavior. but I think having license as required field and undefined value works the same way